conf/layer.conf

```diff
@@ -35,6 +35,7 @@ LICENSE_FLAGS_WHITELIST += "commercial_gstreamer1.0-plugins-ugly"
 # WPE on rpi needs the OpenMAX gstreamer plugin
 LICENSE_FLAGS_WHITELIST += "commercial_libomxil"
 LICENSE_FLAGS_WHITELIST += "commercial_gstreamer1.0-omx"
+LICENSE_FLAGS_WHITELIST += "commercial_gstreamer1.0-libav"
```

Comment:
This can be white listed conditionally under "vc4graphics" MACHINE_FEATURES.

Reply:
that will be better but as it is its ok as well.